prefpicker/templates/schema.json

```
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "prefpicker template schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "pref": {
+      "type": "object",
+      "patternProperties": {
+        "^.*$": {
+          "additionalProperties": false,
+          "type": "object",
+          "properties": {
+            "variants": {
+              "type": "object",
+              "patternProperties": {
+                "^.*$": {
+                  "type": "array"
```

Should exclude object/array as valid values?

```suggestion
                  "type": "array",
                  "items": {
                    "type": [
                      "null",
                      "boolean",
                      "number",
                      "string"
                    ]
                  }
```